pkg/routing/route.go

```diff
@@ -76,6 +76,21 @@ func (br *BidirectionalRoute) Check() error {
 	return nil
 }
 
+// String implements fmt.Stringer
+func (br *BidirectionalRoute) String() string {
+	m := map[string]interface{}{
+		"descriptor": br.Desc.String(),
+		"keep_alive": br.KeepAlive.String(),
+		"fwd_hops":   br.Forward,
+		"rev_hops":   br.Reverse,
+	}
```

Comment:
Would you mind adding an empty line after this line and before `return` statement?